src/BankAccount1.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * Bank Account component with primary methods. (Note: by package-wide
+ * convention, all references are non-null.)
+ *
+ * @mathsubtypes <pre>
+ * BANK_ACCOUNT is (
+ *   money: double(non-negaative number),
+ *   FirstName: string
+ *   LastName: string
+ *   password: int(non-negative)
+ *   exemplar ba
+ *   constraint ba.money >= 0
+ *)
+ *
+ * </pre>
+ * @mathmodel type BankAccountKernel is modeled by BANK_ACCOUNT
+ *
+ * @initially <pre>
+ * ():
+ *  ensures
+ *   this = (0,"","",0)
+ * (double money, String firstName, String lastName, int password):
+ *  requires
+ *   money >= 0
+ *   firstName is not null
+ *   lastName is not null
+ *   password has 6 to 8 digit
+ *  ensures
+ *   this.money = money
+ *   this.firstName =firstName
+ *   this.lastName =lastName
+ *   this.password = password
+ * </pre>
+ */
+public class BankAccount1 extends BankAccountSecondary {
+
+    /**
+     * Representation of {@code this}.
+     */
+    private double money;
+
+    private String firstName;
+
+    private String lastName;
+
+    private int password;
+
+    /**
+     * Creator of initial representation.
+     */
+    private void createNewRep() {
+
+        // TODO - fill in body
+        this.money = 0;
+        this.firstName = null;
+        this.lastName = null;
+        this.password = 0;
+
+    }
+
+    /*
+     * Constructors -----------------------------------------------------------
+     */
+
+    public BankAccount1(String FirstName, String LastName, int password) {
```

Comment:
```suggestion
    public BankAccount1(String firstName, String lastName, int password) {
```

By convention, we tend to start variables with lowercase letters.